pallets/did/src/lib.rs

```diff
@@ -211,6 +211,15 @@ pub enum OperationError {
 	InvalidNonce,
 }
 
+// Error generated when invalid URLs are used as encoded URLs.
```

Comment:
Should probably be a doc comment?

```suggestion
/// Error generated when invalid URLs are used as encoded URLs.
```